### DISCUSSION TITLE
delete elements event

### DISCUSSION THREAD

**Discussion by mtksugi** [bcakmakoglu/vue-flow#795] [Q&A] [answered]
Thank you for all your help.

I want an elements count（`getElements.value.Length`） after delete an element. 

I implement this inner `onNodesChange, onEdgesChange` event, but in `onNodesChange, onEdgesChange`, `getElements` returns the elements before deletion. 

How do I get elements after deletion? 
Thanks.

**Comment by bcakmakoglu** [comment, marked as answer]
You could use a simple watcher to track the count of your elements.

```ts
watch(getElements, (newEls, oldEls) => {
   if (newEls.length !== oldEls.length && newEls.length < oldEls.length) {
      console.log('some element was deleted')
   }
})

**Reply by mtksugi** [reply]
I see.
Thank you very match.
